### ISSUE TITLE
ie11 invalid calling object

### ISSUE THREAD

**Issue by ThieryMichel** [JedWatson/react-input-autosize#29]
I get an Invalid Calling Object error on IE11 on line 67:
```
queueUpdateInputWidth: function queueUpdateInputWidth() {
	nextFrame(this.updateInputWidth);
}
```
After debugging I found that nextFrame equal to `requestAnimationFrame`, and binding nextFrame to window correct the issue.
I will open a PR to fix this.

**Comment by ThieryMichel** [comment]
Or not.

**Comment by ThieryMichel** [comment]
see #30

**Comment by JedWatson** [comment]
Closed by #30
